modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -106,7 +106,7 @@ class Test_Darknet_layers : public DNNTestLayer
         std::string cfg = findDataFile("dnn/darknet/" + name + ".cfg");
         std::string model = "";
         if (hasWeights)
-            model = findDataFile("dnn/darknet/" + name + ".weights", false);
+            model = findDataFile("dnn/darknet/" + name + ".weights", required);
```

Comment:
Lets remove unclear `required` parameter (with meaning of `requiredWeights`).
Just remove `", false"` from here (weights are not "optional" testdata - they are stored in "opencv_extra").

Reply:
I agree, lest just emit the last argument (so make it `true` by default value):

```
            model = findDataFile("dnn/darknet/" + name + ".weights");
```